### PR TITLE
Add NS as Harbor adopter

### PR DESCRIPTION
**Pull request by slushysnowman**
[Related PR in website](https://github.com/goharbor/website/pull/473)

### CODE REVIEW

**Review by OrlinVasilev — APPROVED** [review]
Approved, do not merge until PR on the website is merged

**Review by wy65701436 — APPROVED** [review]
lgtm

**Pull request merged**
